contrib/nydusify/pkg/checker/tool/nydusd.go

```diff
@@ -162,6 +162,7 @@ func (nydusd *Nydusd) Mount() error {
 	nydusd.Umount(true)
 
 	args := []string{
+		"fuse",
```

Comment:
Please revert this change, we are switching to subcommand form and deprecating new subcommand form